### PR TITLE
CXP-654 CXP-650 CPX-652 - New connect space

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**
Jira tickets: 
- CXP-654 
- CXP-650 
- CPX-652

**Renaming**
- [x] Menu System => Menu Connect
- [x] Sub menu Connections => Sub menu Connection settings
- [x] Connection dashboard => Data flows dashboard

**New menu**
- [x] New menu entry

**Screenshots update - Connectivity only**
- [x]  connection-dashboard.png ⇒ new header : _to be done_
- [x]  connection-download-logs-button.png ⇒ header and breadcrumb
- [x]  connection-monitoring-page.png ⇒ header, breadcrumb + menu
- [x]  connections-overview.png ⇒ breadcrumb
- [x]  delete-a-connection.png ⇒ breadcrumb
- [x]  event-log-ui.gif ⇒ header, breadcrumb + menu

### CODE REVIEW

File: content/md/connectivity/connection-dashboard.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ related: what-is-a-connection, how-to-connect-my-pim, manage-your-connections
 
 # Overview
 
-In the PIM, there is a dashboard, called the `Connection dashboard`, to help you follow your [connections](what-is-a-connection.html) activity. You can access it by clicking on the `Activity` menu, then on the `Connection dashboard` submenu. And here you are! :tada:
+In the PIM, there is a dashboard, called `Data flows`, to help you follow your [connections](what-is-a-connection.html) activity. You can access it by clicking on the `Connect` menu, then on the `Data flows` submenu. And here you are! :tada:
```
Comment (camillebourjade) [inline content/md/connectivity/connection-dashboard.md:12]:
```suggestion
In the PIM, there is a dashboard called `Data flows`, to help you follow your [connections](what-is-a-connection.html) activity. You can access it by clicking on the `Connect` menu, then on the `Data flows` submenu. And here you are! :tada:
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 2]
```diff
@@ -7,23 +7,23 @@ ee-only: false
 related: what-is-a-connection, manage-your-connections, connection-dashboard, what-is-an-event-subscription
 ---
 
-# Meet the "Connections"
+# Meet the "Connect" menu
 
-A PIM without any third party connection is pretty useless, right?
+You need to gather all your product data, and this is why you use the Akeneo PIM. But, you also need to retrieve product data or spread them, right?
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:12]:
```suggestion
You need to gather all your product data, and this is why you use Akeneo PIM. But, you also need to retrieve product data or share them, right?
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 3]
```diff
@@ -7,23 +7,23 @@ ee-only: false
...
 
-Indeed, for your PIM to properly work, you first need to connect it to the data flows that are coming from your ERP, DAM or even MDM.  
-Once your products are enriched into the PIM, you then need them to be delivered to all your channels, being syndication, e-commerce or publishing platforms.  
-Maybe, you also need to connect your PIM to a translation tool to enrich your data even more efficiently. One can also think to connect the PIM to some BI tools, in order to get reports on the enrichment workload.
+Indeed, for your PIM to work correctly, you first need to connect it to the data flows that are coming from your ERP, DAM, or even MDM.  
+Once your products are enriched into the PIM, you need them to be delivered to all your channels, such as syndication, e-commerce, or publishing platforms.  
+Maybe, you also need to connect your PIM to a translation tool to enrich your data more efficiently. One can also consider combining the PIM with some BI tools to get reports on the enrichment workload.
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:16]:
```suggestion
Maybe, you also need to connect your PIM to a translation tool to enrich your data more efficiently. You can also consider combining the PIM with some BI tools to get reports on the enrichment workload.
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 4]
```diff
@@ -7,23 +7,23 @@ ee-only: false
...
 
-In the end, all those data flows represent a great deal of connections around your PIM.
+In the end, all those data flows represent plenty of connections around your PIM.
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:18]:
```suggestion
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 5]
```diff
@@ -7,23 +7,23 @@ ee-only: false
...
 
-In the PIM, there is a concept that will help you connect any third party you might think of, to boost the powers of your PIM. It is called the `Connections`. :tada:
+In the PIM, there is a concept that will help you connect any third party you might think of to boost the powers of your PIM. It is called the `Connections`. :tada:
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:20]:
```suggestion
In the PIM, there is a concept that helps you connect any third party you might think of to boost the powers of your PIM. It is called `Connections`. :tada:
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 6]
```diff
@@ -34,37 +34,41 @@ You have landed in the right place! Here are the 3 crucial steps for a good conn
...
-So, your first reflex will be to check if there is an API-based connector on the marketplace for your needs.
+Thanks to our community and partners, we have plenty of extensions ready to be activated on your PIM. To find them, go to your PIM `Connect` menu, click on the `Marketplace` sub-menu, and here you are! 
 
-::: warning
-Your connector should be API-based in order to be compatible with our connections. Want to know why you need to use the API to connect your PIM? The answer is right [here](#why-should-i-use-the-api).
-:::
+The PIM Marketplace page allows you to access our Marketplace and display all the extensions compatible with your PIM.
+For example, if you want to connect your PIM to Magento 2, we provide the [Akeneo connector](https://help.akeneo.com/magento2-connector/index.html).
+
+So, your first reflex will be to check if there is an existing extension on the marketplace for your needs.
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:45]:
```suggestion
So, your first reflex should be to check if there is already an existing extension on the marketplace for your needs.
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 7]
```diff
@@ -34,37 +34,41 @@ You have landed in the right place! Here are the 3 crucial steps for a good conn
...
 
-There may be some third parties for which no API-based connector was created and available on our marketplace. In this case, you will have to develop it to cover your own needs.  
+There may be some third parties for which no API-based extension was created and available on our marketplace. In this case, you will have to develop it to cover your own needs.  
 We provide guides to help you in this task. For example, here is how you can [connect to a DAM](https://api.akeneo.com/guides/dam-connection/introduction.html) with our API. If you can't find a guide covering your use case, we recommend you to get familiar with our [API documentation](https://api.akeneo.com).
 
-## _Step 3_ | Use the credentials in the connector
-That's the easiest part. When you created your connection, the PIM provided you with a set of credentials: a client id, a secret, a username and a password. Put those credentials into your connector. It will be used to authenticate your API calls.
+::: tips
+We publish new extensions regularly, so don't forget to check our [marketplace](https://marketplace.akeneo.com/) regularly! :wink:
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:51]:
```suggestion
We publish new extensions regularly, so don't forget to keep an eye on our [marketplace](https://marketplace.akeneo.com/)! :wink:
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 8]
```diff
@@ -34,37 +34,41 @@ You have landed in the right place! Here are the 3 crucial steps for a good conn
...
+:::
+
+## _Step 3_ | Use the credentials in the extension
+That's the easiest part. When you created your connection, the PIM provided you with a set of credentials: a client id, a secret, a username and a password. Put those credentials into your extension. It will be used to authenticate your API calls.
 
 And... that's it! :)
 
-You can now enjoy your connection. By the way, you will surely need to monitor what is happening on this data flow. There is a feature for that. It is called `Connection dashboard`. Eager to know more? [Click here!](connection-dashboard.html)
+You can now enjoy your connection. By the way, you will surely need to monitor what is happening on this data flow. There is a feature for that. It is called `Data flows`. Eager to know more? [Click here!](connection-dashboard.html)
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:59]:
```suggestion
You can now enjoy your connection. To monitor how your fresh connection is working, you can go to `Data flows`. Eager to know more? [Click here!](connection-dashboard.html)
```

File: content/md/connectivity/how-to-connect-my-pim.md [hunk 9]
```diff
@@ -34,37 +34,41 @@ You have landed in the right place! Here are the 3 crucial steps for a good conn
...
 2. We worked a lot to improve the **performances** of the API. For example, you will experience a better speed when you export product through the API than if you use our basic CSV export stack.
-3. This is the **only way to connect the PIM for our Serenity customers**. Indeed, the Serinity offer only provides a SaaS platform, on which you cannot customize anything. As a consequence, the API is your one and only way to connect the PIM. You can use it to create entire workflows around your PIM, that will create products or request them, depending on your needs.
+3. This is the **only way to connect the PIM for our Serenity customers**. Indeed, the Serenity offer only provides a SaaS platform, on which you cannot customize anything. As a consequence, the API is your one and only way to connect the PIM. You can use it to create entire workflows around your PIM, that will create products or request them, depending on your needs.
```
Comment (camillebourjade) [inline content/md/connectivity/how-to-connect-my-pim.md:71]:
```suggestion
3. This is the **only way to connect the PIM for our SaaS customers**. You can use it to create entire workflows around your PIM, that will create products or request them, depending on your needs.
```

File: content/md/connectivity/manage-your-connections.md [hunk 10]
```diff
@@ -39,8 +39,8 @@ You can create up to 50 connections. :rocket:
 :::
 
 Here are the simple steps to create a connection:
-1. Click on the `System` menu.
-2. Click on the `Connections` submenu.
+1. Click on the `Connect` menu.
+2. Click on the `Connection settings` submenu.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:43]:
```suggestion
2. Then on `Connection settings`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 11]
```diff
@@ -39,8 +39,8 @@ You can create up to 50 connections. :rocket:
...
 3. Click on the `Create` button.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:44]:
```suggestion
3. Click on `Create`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 12]
```diff
@@ -60,7 +60,7 @@ When you create or update a connection, you have to define a "flow type". But wh
 
 The flow type is a central concept in the connection notion. It allows you to characterize the data flows that will interact with your PIM. More precisely it allows to indicate the **direction** of a given flow.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:61]:
```suggestion
The flow type is a central concept in the connection notion. It allows you to characterize the data flows that will interact with your PIM. More precisely it allows indicating the **direction** of a given flow.
```

File: content/md/connectivity/manage-your-connections.md [hunk 13]
```diff
@@ -72,7 +72,7 @@ Choose this option for you connection whenever it represents a data flow enterin
 With this flow type, you can connect ERP, DAM, MDM,...
 :::
 
-If you choose this option, the connection dashboard feature will focus on the data that is pushed inside the PIM via this connection.
+If you choose this option, the Data flows dashboard feature will focus on the data that is pushed inside the PIM via this connection.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:75]:
```suggestion
If you choose this option, the Data flows dashboard will focus on the data that is pushed inside the PIM via this connection.
```

File: content/md/connectivity/manage-your-connections.md [hunk 14]
```diff
@@ -82,7 +82,7 @@ Choose this option for you connection whenever it represents a data flow that is
 With this flow type, you can connect e-commerce platforms, publishing and syndication tools...
 :::
 
-If you choose this option, the connection dashboard feature will focus on the data that is pushed outside the PIM via this connection.
+If you choose this option, the Data flows dashboard feature will focus on the data that is pushed outside the PIM via this connection.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:85]:
```suggestion
If you choose this option, the Data flows dashboard will focus on the data that is pushed outside the PIM via this connection.
```

File: content/md/connectivity/manage-your-connections.md [hunk 15]
```diff
@@ -99,8 +99,8 @@ That being said, even in this case, we advise you to choose between the two opti
 
 # Enable the tracking
 
-For each connection, you can choose whether you want to track it in the [connection dashboard](connection-dashboard.html).  
-To do so, select the `Track in the connection dashboard` checkbox, in the connection setting screen of the connection you want to track.
+For each connection, you can choose whether you want to track it in the [Data flows dashboard](connection-dashboard.html).
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:102]:
```suggestion
For each connection, you can choose whether you want to track it in [Data flows](connection-dashboard.html).  
```

File: content/md/connectivity/manage-your-connections.md [hunk 16]
```diff
@@ -99,8 +99,8 @@ That being said, even in this case, we advise you to choose between the two opti
 
 # Enable the tracking
 
-For each connection, you can choose whether you want to track it in the [connection dashboard](connection-dashboard.html).  
-To do so, select the `Track in the connection dashboard` checkbox, in the connection setting screen of the connection you want to track.
+For each connection, you can choose whether you want to track it in the [Data flows dashboard](connection-dashboard.html).  
+To do so, select the `Track this connection in the dashboard` checkbox, in the connection setting screen of the connection you want to track.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:103]:
```suggestion
To do so, check `Track this connection in the dashboard`, in the connection setting screen of the connection you want to track.
```

File: content/md/connectivity/manage-your-connections.md [hunk 17]
```diff
@@ -163,8 +163,8 @@ Along with the secret revocation, the PIM will automatically regenerate a new se
 :::
 
 Here are the simple steps to go through to revoke your secret:
-1. Click on the `System` menu.
-2. Click on the `Connections` submenu.
+1. Click on the `Connect` menu.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:166]:
```suggestion
1. Click on `Connect`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 18]
```diff
@@ -163,8 +163,8 @@ Along with the secret revocation, the PIM will automatically regenerate a new se
...
+2. Click on the `Connection settings` submenu.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:167]:
```suggestion
2. Click on `Connection settings`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 19]
```diff
@@ -163,8 +163,8 @@ Along with the secret revocation, the PIM will automatically regenerate a new se
...
 3. Click on the connection for which you want to revoke the secret.
 4. In the `Credentials` section, on the right side of the `Secret` line, click on the regenerate icon button.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:169]:
```suggestion
4. In the `Credentials` section, on the right side of the `Secret` line, click on the regenerate icon.
```

File: content/md/connectivity/manage-your-connections.md [hunk 20]
```diff
@@ -187,8 +187,8 @@ Don't really know how the authentication via API works? [Don't worry, there is d
 The PIM shows you the connection password only once, at the connection creation. In case you may have forgotten it, you can regenerate a new one.  
 
 Here's how:
-1. Click on the `System` menu.
-2. Click on the `Connections` submenu.
+1. Click on the `Connect` menu.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:190]:
```suggestion
1. Click on `Connect`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 21]
```diff
@@ -187,8 +187,8 @@ Don't really know how the authentication via API works? [Don't worry, there is d
...
+2. Click on the `Connection settings` submenu.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:191]:
```suggestion
2. Click on `Connection settings`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 22]
```diff
@@ -187,8 +187,8 @@ Don't really know how the authentication via API works? [Don't worry, there is d
...
 3. Click on the connection for which you want to revoke the password.
 4. In the `Credentials` section, on the right side of the `Password` line, click on the regenerate icon button.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:193]:
```suggestion
4. In the `Credentials` section, on the right side of the `Password` line, click on the regenerate icon.
```

File: content/md/connectivity/manage-your-connections.md [hunk 23]
```diff
@@ -233,7 +233,9 @@ To create a user role and link it to your connection:
 5. Click on the `Web API permissions` tab and select `Overall Web API access`.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:233]:
```suggestion
5. Click on `Web API permissions` and select `Overall Web API access`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 24]
```diff
@@ -233,7 +233,9 @@ To create a user role and link it to your connection:
...
 6. You can also select other accesses in the list below depending on what you want your connection to be able to achieve.
 7. Save your configuration by clicking on the `Save` button.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:235]:
```suggestion
7. Save your configuration by clicking on `Save`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 25]
```diff
@@ -270,14 +274,14 @@ To be able to view the catalog, you will need to give permissions to your connec
...
+2. Click on the `Connection settings` submenu.
 3. Click on the connection you want to delete.
 4. Click on the `...` in the top right corner of your screen.
 5. Click on the `Delete` option.
 ![Delete a connection](../img/delete-a-connection.png)
 6. A popin asks you for confirmation. Click on the `Delete` button.
 
 As a result, your connection will be deleted. It means that: 
-- you won't be able anymore to follow its data flows inside the [connection dashboard](connection-dashboard.html),
+- you won't be able anymore to follow its data flows inside the [Data flows dashboard](connection-dashboard.html),
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:286]:
```suggestion
- you will no longer be able to follow its data flows inside the [Data flows dashboard](connection-dashboard.html),
```

**Review by camillebourjade — COMMENTED** [review]
English fix (finish 'the' 'button')

File: content/md/connectivity/manage-your-connections.md [hunk 26]
```diff
@@ -230,10 +230,12 @@ To create a user role and link it to your connection:
...
 3. Click on the `Create role` button.
 4. In the form that appears, give a name to your user role, `My Magento user role` for example.
-5. Click on the `Web API permissions` tab and select `Overall Web API access`.
+5. Click on `Web API permissions` and select `Overall Web API access`.
 6. You can also select other accesses in the list below depending on what you want your connection to be able to achieve.
-7. Save your configuration by clicking on the `Save` button.
-2. Click on the `Connections` submenu.
+7. Save your configuration by clicking on `Save`.
+
+1. Click on the `Connect` menu.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:237]:
```suggestion
1. Click on `Connect`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 27]
```diff
@@ -230,10 +230,12 @@ To create a user role and link it to your connection:
...
+2. Click on the `Connection settings` submenu.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:238]:
```suggestion
2. Click on `Connection settings`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 28]
```diff
@@ -254,8 +256,10 @@ To create a user group and link it to your connection:
 2. Click on the `User groups` submenu.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:256]:
```suggestion
2. Click on `User groups`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 29]
```diff
@@ -254,8 +256,10 @@ To create a user group and link it to your connection:
...
 3. Click on the `Create group` button.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:257]:
```suggestion
3. Click on `Create group`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 30]
```diff
@@ -254,8 +256,10 @@ To create a user group and link it to your connection:
...
 4. In the form that appears, give a name to your user group, `My Magento user group` for example.
-7. Save by clicking on the `Save` button.
-2. Click on the `Connections` submenu.
+5. Save by clicking on the `Save` button.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:259]:
```suggestion
5. Click on `Save`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 31]
```diff
@@ -254,8 +256,10 @@ To create a user group and link it to your connection:
...
+
+1. Click on the `Connect` menu.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:261]:
```suggestion
1. Click on `Connect`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 32]
```diff
@@ -254,8 +256,10 @@ To create a user group and link it to your connection:
...
+2. Click on the `Connection settings` submenu.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:262]:
```suggestion
2. Click on `Connection settings`.
```

File: content/md/connectivity/manage-your-connections.md [hunk 33]
```diff
@@ -270,14 +274,14 @@ To be able to view the catalog, you will need to give permissions to your connec
 
 If you created a connection you don't want to use anymore, you can delete it.
 
-1. Click on the `System` menu.
-2. Click on the `Connections` submenu.
+1. Click on the `Connect` menu.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-connections.md:277]:
```suggestion
1. Click on `Connect`.
```

**Review by camillebourjade — APPROVED** [review]

**Pull request merged**